features/integration/compile.feature

```diff
@@ -10,3 +10,26 @@ Feature: Compile
       | program                 | status | result     | hash                                                         |
       | "programs/one.teal"     | 200    | "AiABASI=" | "YOE6C22GHCTKAN3HU4SE5PGIPN5UKXAJTXCQUPJ3KKF5HOAH646MKKCPDA" |
       | "programs/invalid.teal" | 400    | ""         | ""                                                           |
+
+
```

Comment:
In theory (and in my own practice in the Python SDK) it is possible to omit the "*.tok" TEAL binary, and have `algod` do the compilation for you. This makes it a bit easier to write/tweak TEAL to make it feat the integration tests. But unveiling this ability would require some additional support in the other SDK's. So it's worth further discussion.